gpflow/models/gpmc.py

```diff
@@ -47,8 +47,8 @@ def __init__(self, X, Y, kern, likelihood,
             L L^T = K
 
         """
-        X = DataHolder(X)
-        Y = DataHolder(Y)
+        X = DataHolder(X, check_ndim_is=2)
```

Comment:
We've already discussed it and decided that this change shouldn't be in any Parameter, but in the model itself.

Reply:
Yeah... that's what I did? I changed it in the model?